checks/memoryLimitsMissing.yaml

```diff
@@ -25,3 +25,7 @@ schema:
               type: string
               not:
                 const: ''
+mutations:
+  - op: add
+    path: /resources/limits
+    value: {memory: "512Mi"}
```

Comment:
Oh wait I get it now. Why are these different from CPU?

Reply:
@makoscafee any reason these are different? I think this check would modify `limits.cpu` as-is

Reply:
🤔 maybe we need two mutations: one to add `limits`, and then one to add `limits.memory`?

Reply:
Done